platform/ontology/client.go

```diff
@@ -16,19 +17,38 @@ const (
 )
 
 func (c *Client) GetTxsOfAddress(address, assetName string) (txPage *TxPage, err error) {
-	url := fmt.Sprintf("address/%s/%s/%d/1", address, assetName, TxPerPage)
+	url := fmt.Sprintf("/api/v1/explorer/address/%s/%s/%d/1", address, assetName, TxPerPage)
 	err = c.Get(&txPage, url, nil)
 	return
 }
 
 func (c *Client) CurrentBlockNumber() (response *BlockResults, err error) {
-	url := fmt.Sprintf("blocklist/%d", requestOnlyLatestBlockAmount)
+	url := fmt.Sprintf("/api/v1/explorer/blocklist/%d", requestOnlyLatestBlockAmount)
 	err = c.Get(&response, url, nil)
+	if int(response.Error) != 0 {
+		err = errors.E("explorer client CurrentBlockNumber", errors.Params{"platform": "ONT"})
+	}
 	return
 }
 
 func (c *Client) GetBlockByNumber(num int64) (block *BlockResult, err error) {
-	url := fmt.Sprintf("block/%d", num)
+	url := fmt.Sprintf("/api/v1/explorer/block/%d", num)
 	err = c.Get(&block, url, nil)
+	if int(block.Error) != 0 {
+		err = errors.E("explorer client GetBlockByNumber", errors.Params{"platform": "ONT"})
+	}
+	return
+}
+
+func (c *Client) GetTxDetailsByHash(hash string) (tx TxV2, err error) {
+	url := fmt.Sprintf("/v2/transactions/%s", hash)
+	var txRes TxResponse
+	err = c.Get(&txRes, url, nil)
+	if txRes.Msg != "SUCCESS" {
+		err = errors.E("explorer client GetTxDetailsByHash", errors.Params{"platform": "ONT"})
+	}
+	if int(txRes.Result.EventType) == 3 {
+		tx = txRes.Result
+	}
 	return
```

Comment:
```suggerstion
	if int(txRes.Result.EventType) != 3 {
		return tx, errors.E("GetTxDetailsByHash invalid tx type", errors.Params{"platform": "ONT", "hash": hash, "hash": txRes.Result.EventType})
	}
	return
```

Reply:
Nope, there is normal behavior when we will get multiple tx Types, we do not need to return an error because of it